ang/crmCaseType.js

```diff
@@ -169,7 +178,10 @@
         $scope.caseType = apiCalls.caseType;
       } else {
         // new case type
+        var defaultCategory = _.find($scope.caseTypeCategories, { name: 'WORKFLOW' }) || {};
```

Comment:
Why `WORKFLOW` is selected as default?

Reply:
When editing, for older values which does not have category saved, should not we assign the default value there too?

Reply:
The default being workflow is part of the requirement. It's supposed to be the more generic one, where as vacancy is more specific.

Older case types will have a category defined using the PHP upgrader, so there's no need to handle them. See here:

https://github.com/compucorp/civicrm-core/pull/15/files#diff-8051ac48f8f65e5e07d99d757afccf4cR169